spec/ConvertGobstonesAstToMulangAstSpec.hs

```diff
@@ -0,0 +1,36 @@
+module ConvertGobstonesAstToMulangAstSpec (spec) where
+
+import           Test.Hspec
+import           Language.Mulang
+import			 Language.Mulang.Parsers.Gobstones
+
+spec :: Spec
+spec = do
+
+describe "translateProgramGobstonesToMulangExpression" $ do
+    it "translate simple program Gobstones" $ do
+      let gobstonesAst =  parseGobstones "[{\"alias\": \"program\",\"body\": null,\"from\": 0}]"
+
+      gobstonesAst `shouldBe` Sequence [MuNull]
```

Comment:
It should be just `MuNull`: it represents an empty set of expressions, while Sequence only is necessary when you have 2 or more expressions.

Please have a look at `Builder` package, which can convert "wrong" trees like `Sequence []` into `MuNull`